aries-site/src/pages/index.js

```diff
@@ -10,20 +10,7 @@ const pageDetails = getPageDetails(title);
 const cards = getCards();
 
 const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/carte-cards-dark.svg', light: '/carte-cards.svg' },
-      alt: 'HPE Carte Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useTiles: true,
-    }}
-    title={title}
-    isLanding
-  >
+  <Layout backgroundImage={pageDetails.backgroundImage} title={title} isLanding>
```

Comment:
I am getting a console warning related to the backgroundImage when I load a page in light mode. It seems as if the staticProps are being initialized in one themeMode on the Server, but the Client receives a different Prop

 "index.js:1 Warning: Prop `src` did not match. Server: "/carte-cards-dark.svg" Client: "/carte-cards.svg""

![Screen Shot 2020-04-08 at 2 54 43 PM](https://user-images.githubusercontent.com/1756948/78832682-e748dc80-79a8-11ea-9ad0-1a095c32914a.png)

Reply:
Do we need to modify the `getStaticProps` function in `[topic].js` to accommodate for these assets which are dependent on `themeMode`?

Reply:
Disregard. Looks like this error is on Master and unrelated to the work here.